packages/tempo/doc/tempo.ticker.md

```
@@ -118,16 +178,28 @@ using countdown = Tempo.ticker({ seconds: -1, seed: "00:00:10" }, (t, stop) => {
 ## API Reference
 
 ### `Tempo.ticker(arg1?: number | string | bigint | TickerOptions, arg2?: Callback): TickerResult`
-Creates a reactive stream of `Tempo` instances at regular intervals. Defaults to a **1-second** interval if omitted.
+Creates a reactive stream of `Tempo` instances at regular intervals. Defaults to a **1-second** interval if no duration or interval properties are specified.
 
 #### `TickerOptions`
 | Property | Type | Description |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)